index.php

```diff
@@ -15,6 +15,14 @@
     <meta name="viewport" content="width=device-width">
 </head>
 <body>
-    <?php echo displayArray($resultArray); ?>
+    <header>
+        <p class="title">CLASSIC</p>
```

Comment:
why did you make your title all separate p tags?

Reply:
To enable flex wrap and to space the words more stylistically